### PR TITLE
Skip file copying when the item inside the output dictionary is empty

### PR DESCRIPTION
**Pull request by junyk**

### CODE REVIEW

**Comment by junyk** [comment]
@rlupat @illusional 
Hey both, since this is related to Janis assistant, just want to double check if this change makes sense:
The reason why I added this is because cromwell and cwltool final output are different for when our PythonCodeTool ParseFastqcAdaptors return empty values

```
# cwltool
2021-03-29T14:32:39 [INFO]: engine_outputs
2021-03-29T14:32:39 [INFO]: {}

# cromwell
2021-03-29T14:36:22 [INFO]: engine_outputs
2021-03-29T14:36:22 [INFO]: {'adaptor_sequences': []}
```

So, without adding this extra check, we will get an error at this line https://github.com/PMCC-BioinformaticsCore/janis-assistant/blob/09ad5466a1dd398ceebfdf17bd237fd6e5ee6009/janis_assistant/management/workflowmanager.py#L1464
because `eout` is empty.

File: janis_assistant/management/workflowmanager.py [hunk 1]
```diff
@@ -1431,6 +1431,13 @@ def copy_outputs_if_required(self):
                     f"Couldn't find expected output with tag {out.id_}, found outputs ({', '.join(eoutkeys)}"
                 )
                 continue
+
+            # not None but just some empty values like an empty list or empty string
+            if not eout:
+                return Logger.critical(
+                    f"Engine '{self.engine.id()}' didn't return {out.id_} outputs for Janis to copy, skipping"
+                )
+
```
Comment (junyk) [inline janis_assistant/management/workflowmanager.py:1440]: so similar to what we already did a step earlier https://github.com/PMCC-BioinformaticsCore/janis-assistant/blob/09ad5466a1dd398ceebfdf17bd237fd6e5ee6009/janis_assistant/management/workflowmanager.py#L1419

But here, we look into each item in the outputs dictionary

**Comment by illusional** [comment]
Seems reasonable to me!

**Comment by junyk** [comment]
@illusional thanks Michael!

**Pull request merged**